addOns/reports/src/main/zapHomeFiles/reports/traditional-html-plus/report.html

```diff
@@ -518,6 +518,116 @@ <h3 th:text="#{report.alerts.detail}">Alert Detail</h3>
 			<div class="spacer"></div>
 		</th:block>
 	</th:block>
+
+	<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+		<th:block
+			th:if="${reportData.reportObjects.get('seqAScanData') != null}">
+			<h3 th:text="#{report.sequences.summary.name}">Summary of
+				Sequences</h3>
+			<sup>For each step: result (Pass/Fail) - risk (of highest
+				alert(s) for the step, if any).</sup>
+
+			<th:block
+				th:each="seq, seqState: ${reportData.reportObjects.get('seqAScanData').seqData}">
+				<table class="alerts">
+					<tr>
+						<td th:text="${seq.sequenceName}">sequenceName</td>
+						<th:block th:each="step, stepState: ${seq.getSteps()}">
+							<td th:class="(${step.isPass()} ? 'pass' : 'fail')"
+								th:with="riskLvl=(${step.getHighestAlert()})"><span
+								th:text="(${step.isPass()} ? 'Pass' : 'Fail')">isPass</span> <span
+								th:if="${riskLvl &gt; 0}"
+								th:class="'alert-' + ${step.getHighestAlert() + 'a'}"
+								th:text="' - '">flag</span> <span
+								th:text="(${riskLvl &gt; 0} ? ${helper.getRiskString(riskLvl)} : '')">risk</span>
+							</td>
+						</th:block>
+					</tr>
+				</table>
+			</th:block>
+
+			<div class="spacer-lg"></div>
+
+			<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+				<th:block
+					th:if="${reportData.reportObjects.get('seqAScanData') != null}">
+					<h3 th:text="#{report.sequences.details.name}">Sequences
+						Details</h3>
+					<sup>With the associated active scan results.</sup>
+
+					<th:block
+						th:each="seq, seqState: ${reportData.reportObjects.get('seqAScanData').seqData}">
+						<h4 th:text="${seq.sequenceName}">sequenceName</h4>
+						<div class="lm2">
+							<th:block th:each="step, stepState: ${seq.getSteps()}">
+
+								<h4
+									th:text="'Step ' + ${stepState.index + 1} + ': ' + ${step.getOriginalMsg().getRequestHeader().getMethod()} + ' - ' + ${step.getOriginalMsg().getRequestHeader().getURI().toString()}">stepDesc</h4>
```

Comment:
Could be a real link?

Reply:
I don't see a way to do that. I tried concatenating in the anchor tags as well but it's escaped when rendered (likely to prevent XSS which makes sense).

Reply:
You'll need to split up the elements. Something like `<h4><th:block th:text="#{..}"></th:block><a th:href="${uri}" th:text="${uri}" href="url.html">URL</a></h4>`